phase1-coordinator/src/objects/chunk.rs

```diff
@@ -335,27 +341,27 @@ impl Chunk {
     /// If the operations succeed, returns `Ok(())`. Otherwise, returns `CoordinatorError`.
     ///
     #[inline]
-    pub(crate) fn add_contribution(
+    pub fn add_contribution(
         &mut self,
         contribution_id: u64,
-        participant: &Participant,
+        contributor: &Participant,
         contributed_locator: String,
         contributed_signature_locator: String,
     ) -> Result<(), CoordinatorError> {
         // Check that the participant is a contributor.
-        if !participant.is_contributor() {
+        if !contributor.is_contributor() {
```

Comment:
such spelling looks fun, but maybe we can avoid such checks in the future after switching to newtypes